imgur-screenshot

```diff
@@ -52,6 +52,13 @@ load_default_config() {
     declare -g SCREENSHOT_FULL_COMMAND="screencapture %img"
     declare -g OPEN_COMMAND="open %url"
     declare -g CLIPBOARD_COMMAND="pbcopy"
+  elif is_wayland; then
+    declare -g SCREENSHOT_SELECT_COMMAND="slurp_select %img"
```

Comment:
I'd rather have the command in here instead of introducing a new, command-specific function below:

```suggestion
    declare -g SCREENSHOT_SELECT_COMMAND='grim -g "$(slurp)" %img'
```

Reply:
It's not working because bash adds some quotes in the regexp.
I was tried it before open the pr.